scripts/create-module.js

```diff
@@ -0,0 +1,50 @@
+const { execSync } = require('child_process');
```

Comment:
move this to the `commands` folder and follow our CLI pattern

Reply:
@AHS12  I have made the changes as you specified.